### ISSUE TITLE
Extension host crashed

### ISSUE THREAD

**Issue by madhusudanbabar** [egonyans/vscode-ext-color-highlight#103]
when i try to import a scss file into another scss file, while this extension enabled, it causes extension host to crash

here's the screenshot
![Screenshot (12)](https://user-images.githubusercontent.com/28552728/83616367-602d7480-a5a5-11ea-802e-a497744e6592.png)

**Comment by madhusudanbabar** [comment]
i tried with vscode insider builds too.

**Comment by tupacan** [comment]
Same problem here.

**Comment by ronilaukkarinen** [comment]
Same here. Any fix?

**Comment by madhusudanbabar** [comment]
nope, I reported it [here](https://github.com/microsoft/vscode/issues/99218) but nothing helpful

**Comment by praveen7557** [comment]
The same thing was happening with me too, So disabled this extension and started using [colorize](https://marketplace.visualstudio.com/items?itemName=kamikillerto.vscode-colorize).

**Comment by VirajInsharp** [comment]
Still not fixed?
